source/MaterialXTest/GenOsl.cpp

```diff
@@ -92,7 +92,7 @@ TEST_CASE("GenShader: OSL Implementation Check", "[genosl]")
     generatorSkipNodeTypes.insert("light");
     mx::StringSet generatorSkipNodeDefs;
 
-    GenShaderUtil::checkImplementations(context, generatorSkipNodeTypes, generatorSkipNodeDefs, 65);
+    GenShaderUtil::checkImplementations(context, generatorSkipNodeTypes, generatorSkipNodeDefs, 65+27);
```

Comment:
Any reason you're writing this as 65+27 instead of 92?

Reply:
Mostly to remember there are 27 tests to add :)